app/serializers/api/v1/customer_serializer.rb

```diff
@@ -16,11 +16,9 @@ class CustomerSerializer < BaseSerializer
         address(object.shipping_address)
       end
 
-      attribute :balance, if: proc { |_record, params|
-        params && params[:display_customer_balance] == true
-      } do |object|
-        { value: object.balance_value, time: object.balance_time }
-      end
+      attribute :balance, if: proc { |record|
```

Comment:
Now we just check if customer responds to balance_value, no params passed as you suggested